test/stablex/batch_exchange.js

```diff
@@ -1822,4 +1823,27 @@ contract("BatchExchange", async accounts => {
       assert.equal(await batchExchange.hasToken.call(erc20_1.address), true)
     })
   })
+  describe("Special Examples", async () => {
```

Comment:
nit: maybe call this section regression tests?

Reply:
```suggestion
  describe("Regression Tests", async () => {
```